packages/rpc-core/src/index.ts

```diff
@@ -122,6 +122,8 @@ export class RpcCore implements RpcInterface {
 
   public readonly system!: RpcInterface['system'];
 
+  public readonly web3!: RpcInterface['web3'];
```

Comment:
This should not be needed when adding to the above location, only when a new top-level interface folder is created. (Part of the reason net is included in the eth structure above with an explicit alias - apart from keeping things together consistently)

Reply:
Ok... I was wrong in this... Now that I fixed the generation elsewhere, this is indeed needed. Sorry about that.

Reply:
np ^^